lib/board_viewer.rb~

```diff
@@ -0,0 +1,18 @@
+class BoardViewer
+  def show(board)
+    value = board.regions.first.squares.first.number
+    [[value.nil? ? 0 : 1]]
+  end
+
+  def get_rows(board)
+    board.regions.select { |region| region.type == :row }
+  end  
+
+  def show_row(row)
+    row.squares.map { |square| square.number ? square.number : 0 }
+  end
+  
+  def show(board)
+    get_rows(board).map { |row| show_row(row) }
+  end
+end
```

Comment:
This is a backup file! Make sure these are tidied up before pushing to Github, this can be done in your .gitignore. In fact, add this:

```
[._]*.s[a-w][a-z]
[._]s[a-w][a-z]
*.un~
Session.vim
```